private/buf/cmd/buf/command/alpha/plugin/pluginpush/pluginpush.go

```diff
@@ -205,6 +191,22 @@ func run(
 		nextRevision = 0
 	} else {
 		nextRevision = currentRevision.Revision + 1
+		// Push container if digest changed from current revision
+		if currentRevision.ContainerImageDigest != plugin.ContainerImageDigest() {
```

Comment:
Small optimization but if CI/CD flows run `buf alpha plugin push ...`, this will prevent us pushing up unique tags for the same digest as the last run.